<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindArticeDialog</class>
 <widget class="QDialog" name="FindArticeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поиск товара</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="leSearchString"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tvArticle"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>GroupBox</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbOK">
       <property name="text">
        <string>ОК</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leSearchString</tabstop>
  <tabstop>tvArticle</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>pbOK</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbOK</sender>
   <signal>clicked()</signal>
   <receiver>FindArticeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>FindArticeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
